econml/data/dgps.py

```diff
@@ -93,3 +94,182 @@ def _process_ihdp_sim_data():
     # Append a column of ones as intercept
     X = np.insert(X, 0, np.ones(X.shape[0]), axis=1)
     return T, X
+
+
+class StandardDGP():
```

Comment:
Do we expect users to use this class to test things?  If so, perhaps also add a DGP section to the documentation (and at least add it to the module docs).  But if this is mainly for internal use, add a comment to the docstring indicating that instead.